### PR TITLE
tweet for s1e2

### PR DESCRIPTION
**Pull request by MichaelClifford**
Added tweet for episode 2.

Moved eposide 1 tweet to the correct folder.

### CODE REVIEW

**Comment by MichaelClifford** [comment]
cc @oindrillac @goern

File: tweets/2021-05/s1e2.tweet [hunk 1]
```diff
@@ -0,0 +1 @@
+Wondering where to get started with cloud native data science development? Watch https://youtu.be/iI_-lqi3vP4 as @michaelclifford walks you through getting up and running with your very own data science working environment on the Operate First platform. :)
```
Comment (oindrillac) [inline tweets/2021-05/s1e2.tweet:1]:
```suggestion
Wondering where to get started with cloud native data science development? Watch Episode 2 on https://youtu.be/iI_-lqi3vP4 as @michaelclifford walks you through getting up and running with your very own data science working environment on the Operate First platform. :)
```

**Pull request merged**